src/Domain/DonationRepository.php

```diff
@@ -210,9 +215,12 @@ public function releaseMatchFunds(Donation $donation): void
         // were able to observe. This is why we now go straight to trying to `acquire()`. If
         // this returns false, we are in the contested lock case and know to drop this attempt.
 
-        $fundsReleaseLock = $this->lockFactory->createLock("release-funds-{$donation->getUuid()}");
+        $fundsReleaseLock = $this->getFundsReleaseLock($donation);
 
         try {
+            // We no longer `release()` the lock, holding it for the default 5 mins instead unless a thread newly
```

Comment:
Minor but I think using "no longer" makes it more work to read the code as you're asking the reader to compare what we do now to what we did in a previous version of matchbot.

```suggestion
            // We don't `release()` the lock, holding it for the default 5 mins instead unless a thread newly
```